backend/src/main/java/com/redhat/cloud/notifications/routers/EventResource.java

```diff
@@ -62,8 +62,7 @@ public static class V2 extends EventResource {
     @GET
     @Produces(APPLICATION_JSON)
     @RolesAllowed(RBAC_READ_NOTIFICATIONS_EVENTS)
-    @Operation(summary = "Retrieve the event log entries.", description =
-            "Allowed `sort_by` fields are `bundle`, `application`, `event` and `created`. The ordering can be optionally specified by appending `:asc` or `:desc` to the field, e.g. `bundle:desc`. Defaults to `desc` for the `created` field and to `asc` for all other fields."
+    @Operation(summary = "Retrieve the event log entries", description = "Retrieves the event log entries. Use this endpoint to review a full history of the events related to the tenant. You can sort by the bundle, application, event, and created fields. You can specify the sort order by appending :asc or :desc to the field, for example bundle:desc. Sorting defaults to desc for the created field and to asc for all other fields."
```

Comment:
bundle:desc Sorting. . . .

seems like sorting should be lowercase

Reply:
It's the beginning of the sentence :-)